README.md

```diff
@@ -30,48 +30,92 @@ java -jar app-v-0.1.0.war
 ```
 
 
-<!--
+
  ## Running in Docker
 
+
 To run the application using Docker you have to install Docker and docker-compose on your local computer.  To install Docker see [here](https://docs.docker.com/get-docker/) and to install docker-compose see [here](https://docs.docker.com/compose/install/). 
 
-Assuming you have installed both Docker and docker-compose, to run the Onto Viewer application together with its front-end on your local machine, please make sure that you have the following folder setup:
+Create `onto-viewer-web` and `strapi-dashboard` directories next to onto-viewer root folder.
```

Comment:
These folders need to be added to .gitignore file so that the created folders will not update git